tests/periph_pm/Makefile.board.dep

```diff
@@ -0,0 +1,8 @@
+# For sam0 rtt conflicts with rtc, make will select only rtc because of
+# order of inclusion, but to resolve in the same way as Kconfig this also
+# selects ztimer_no_periph_rtt to select the same modules in Kconfig and make
+ifneq (,$(filter-out sam3,$(filter sam%,$(CPU))))
+  ifneq (,$(filter ztimer%,$(USEMODULE)))
+    USEMODULE += ztimer_no_periph_rtt
```

Comment:
It seems this works and it is commented well enough.  I feel like cpu specific code should not be in the application, but as this is somehow a workaround for the rather complex rtt/ztimer maybe we allow it for now until a better solution is found.

Reply:
`Makefile.board.dep` is there exactly for this reason, sure most of the time its based on `$(BOARD)` but IMO this is exactly the kind of usage. So far we have force `Kconfig` to match `make` weird behaviour, I see no harm in doing it the other way around here. IMO we shouldn't stall this one!

Reply:
Fair enough.  It is documented and everything is passing.  Please squash!